docs/src/man/reshaping_and_pivoting.md

```diff
@@ -244,6 +244,23 @@ julia> first(widedf, 6)
 │ 6   │ setosa        │ 6     │ 1.7         │ 0.4        │ 5.4         │ 3.9        │
 ```
 
+You can even skip the `:variable` and `:value` variables and use:
```

Comment:
What I meant is that we are not skipping the variables but positional arguments in the call. Would this be better?
```suggestion
You can even skip passing the `:variable` and `:value` values as positional arguments, as they will be used by default, and write:
```

Reply:
```suggestion
You can even skip passing the `:variable` and `:value` values as positional arguments, as they will be used by default, and write:
```